operator/pkg/remote/kyma_synchronization_context.go

```diff
@@ -286,3 +290,24 @@ func GetRemoteObjectKey(kyma *v1alpha1.Kyma) client.ObjectKey {
 	}
 	return client.ObjectKey{Namespace: namespace, Name: name}
 }
+
+func (c *KymaSynchronizationContext) InsertWatcherLabels(ctx context.Context, remoteKyma *v1alpha1.Kyma) error {
+	recorder := adapter.RecorderFromContext(ctx)
+
+	ownedByValue := fmt.Sprintf("%s__%s", c.ControlPlaneKyma.Namespace, c.ControlPlaneKyma.Name)
+	watchedByValue := "lifecycle-manager"
+
+	if remoteKyma.Labels == nil {
+		remoteKyma.Labels = make(map[string]string)
+	}
+
+	remoteKyma.Labels["operator.kyma-project.io/owned-by"] = ownedByValue
+	remoteKyma.Labels["operator.kyma-project.io/watched-by"] = watchedByValue
```

Comment:
```suggestion
	remoteKyma.Labels[v1alpha1.ManagedBy] = managedByValue
```

Reply:
This is corresponding to https://github.com/kyma-project/lifecycle-manager/pull/182/files#r970782859